FRPs/proposal-26-security.md

```diff
@@ -0,0 +1,32 @@
+---
+id: 26
+title: Quantifying the Impact of MEV on blockchain consensus security
+team: Liyi Zhou, Kaihua Qin, Arthur Gervais
+created: 2020-12-10
+paper: 2
+---
+
+# Summary
+Multiple works have expressed the qualitative intuition that MEV deteriorates (PoW) blockchain consensus security. Yet, we’re not aware of any quantitative results supporting this hypothesis. The literature proposes various methods on quantifying (PoW) blockchain security. One common insight is that the stale block rate aggravates the risks of double spending and selfish mining. That means, the more forks we can observe on-chain, the easier game becomes for an adversary to defraud other blockchain participants. If MEV encourages miners to fork the blockchain, MEV would directly deteriorate the blockchain’s consensus security. In this FRP we aim to quantitatively reason about the impact of MEV on blockchain security.
+
+## Background and Problem Statement
+In this FRP, we would like to find the minimum MEV value, at which an MEV-aware miner will start to fork the blockchain to claim MEV. If this threshold is very low (e.g., < 10 times the block reward), then MEV clearly is a danger to the chain. Further, we want to extend the study to a multiplayer game, where multiple MEV-aware miners would engage to compete over an MEV opportunity. We then aim to quantify the average and maximum fork length that we can expect to observe.
```

Comment:
Do you have any initial idea on how you'd calculate that? Would you assume that you have a way to observe the ETH value of the available MEV at 1 or more blocks, and compare that against the block reward? The result here sounds straightforward if you assume rational miners, but miners are not always playing rationally for unknown reasons, e.g. they do not selfish mine even if it's the optimal strat because they are structurally long ETH and that kind of activity undermines the quality of their investment

Reply:
There will be a few necessary assumptions, for instance that the adversarial miner knows the average block reward (ETH+regular tx fees+uncles etc), the potential MEV reward, and that the miner is rational. Without rationality I don't see a way to quantify security.
Your example that miners have an incentive not to hurt the ETH value is probably right. If we were able to quantify the extend to which extracting MEV would hurt the ETH price, then we could account this for in a model. But I am not sure this is possible to quantify, mostly because we've seen blockchains with inflation bugs, where the token price nearly wasn't affected (Zerocoin if I recall correctly). As such, it's not clear whether `unconventional` mining behavior does hurt the native blockchain coin price and hence influences the miner's decisions.